CHANGELOG.md

```diff
@@ -3,15 +3,27 @@ Release Date: unreleased
 
 ## RCNet v2 to RCNet v3 Migration Guide
 
-_WORK IN PROGRESS_
+Use Bech32m-encoded transaction hashes in `/transaction/committed-details` and `/transaction/status` endpoints.
+
+TODO TODO TODO TODO
```

Comment:
could we fill it or elaborate what's left to do here?

Reply:
Updated.